README.md

```diff
@@ -1,8 +1,48 @@
+
+**ElectionBlock, Blockchain-Based Voting System**: Collaborated with a team of four Software Engineering students to develop a Python permission-based blockchain voting system that integrates a biometric scanner for authentication and proof of authority as a consensus protocol. Developed in Python, the system runs on its own centralized blockchain network. Some of the technologies used include Flask, Redis (Pub-Sub Broker), React, and Raspberry Pi. Published under the 2021 IEEE International Conference on Software Architecture Companion (ICSA-C). (https://bit.ly/3r6TgmA)
+
+**COVID-19 Tracking Mobile Application**: Collaborated with a team of four Software Engineering students to develop a COVID-19 mobile application using Android (Java), which keeps users up- to-date with the latest COVID-19 statistics. The application includes a main page with updated global statistics, the use of Google Maps API to pinpoint a location and display statistics, and a search query function to display statistics for user-specific countries/cities.
+
+**Polling Cloud Web Application**: Developed a polling web application using Python (Flask), HTML, CSS, Amazon RDS (MySQL) and deployed it to an Amazon EC2 instance (IaaS). The application allows users to create their own polls, vote in any poll made by users, and view results of all polls in a table format. All data is stored and accessible using a MySQL database. The database was setup using Amazon RDS (SaaS).
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2021-present)**: Collaborated in a Scrum team with Agile environment as a Software Developer/Data Engineer. Within Scrum meetings, I assisted others in project related issues and had the opportunity to lead as a scrum leader. Developed numerous projects using GitFlow which incorporate key characteristics and demanding technical skills of a software developer. This includes using technologies and methodologies like Linux, SQL, Docker, Bash, Java, Python, Agile/Scrum, GCP, and more.
+
+**Program Assistant, The Village Children's Program (2017-2021)**: Collaborated with various childhood educators to supervise children aged 4-13, create a safe environment and space for children, ensure the well-being and engagement of children under our supervision, communicated with parents of the children to enable trust and good relationships. I was also given the opportunity to lead and supervise multiple daily physical fitness classes of 30 students. Responsibilities included planning activities for different classes (ex: basketball drills and games, soccer games, children's gym games etc.). 
```

Comment:
**Software Developer, Jarvis (2021-present)**: Collaborated in a Scrum team with Agile environment as a Software Developer/Data Engineer. Within Scrum meetings, I assisted others in project-related issues and had the opportunity to lead as a scrum leader. Developed numerous projects using GitFlow, which incorporate key characteristics and demanding technical skills of a software developer. This includes using technologies and methodologies like Linux, SQL, Docker, Bash, Java, Python, Agile/Scrum, GCP, and more.
		

		**Program Assistant, The Village Children's Program (2017-2021)**: Collaborated with various childhood educators to supervise children aged 4-13, create a safe environment and space for children, ensure the well-being and engagement of children under our supervision, communicate with parents of the children to enable trust and good relationships. I was also given the opportunity to lead and supervise multiple daily physical fitness classes of 30 students. Responsibilities included planning activities for different classes (ex: basketball drills and games, soccer games, children's gym games.